### PR TITLE
PUBDEV-5220: Populate the mapping frame correctly

### PR DESCRIPTION
**Pull request by nikhilshekhar**
This fixes the issue while creating the mapping frame in the Aggregator

### CODE REVIEW

File: h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ public void map(Chunk c1, NewChunk nc) {
           nc.addNum(ArrayUtils.find(keep, gid));
         }
       }
-    }.doAll(exAssignment)._fr.vec(0);
+    }.doAll(Vec.T_NUM,exAssignment).outputFrame().vec(0);
```
Comment (michalkurka) [inline h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java]: How come the test didn't catch that? Do we need to adjust the test then as well?

**Comment by michalkurka** [comment]
LGTM, please adjust the test if necessary.

File: h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java (hunk 1 above)
Comment (nikhilshekhar) [reply h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java]: The frame was getting populated by the wrong frame. The content populated were wrong but the length of frame etc were as expected. So, the tests do not need to change.

Comment (michalkurka) [reply h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java]: Right, the test is not failing. But can we change the test to make sure it catches this type of a bug?

Imagine I would just accidentally overwrite your change. The tests should detect that.

Comment (nikhilshekhar) [reply h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java]: Added a new unit test to check the content of the frame.

Comment (michalkurka) [reply h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java]: @nikhilshekhar thank you!

**Pull request merged**